SEC/Equities/EquityInstruments.rdf

```diff
@@ -406,12 +420,73 @@
 		<skos:definition xml:lang="en">position in an equity instrument</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;EquityRedemptionProvision">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;RedemptionProvision"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasMinimumRedemptionPrice"/>
+				<owl:onClass rdf:resource="&fibo-fbc-fi-ip;MarketPrice"/>
```

Comment:
MarketPrice seems the wrong class here - it has a pricingSource (market) and also a dateTime. Neither of which are appropriate. Why not MonetaryPrice as per the next restriction?

Reply:
We changed it to monetary price to simplify, although in order to redeem the share it is likely required to have evidence from the market, possibly with more than one pricing source.